### ISSUE TITLE
Question: Is BSRNet.pth model, L1 loss trained model?

### ISSUE THREAD

**Issue by vivekalig** [HeaseoChung/BSRGAN-PyTorch#5]
Hello,
I am bit confused with what is BSRNet.pth'' in the 'pre-trained' folder? Is BSRNet.pth the model from initial training with L1 loss only? 
Or, it is something else?
Thank you.

**Comment by HeaseoChung** [comment]
yes, you have to use BSRnet to generate BSRGAN model 

thanks

**Comment by vivekalig** [comment]
Sure. Thank you.
